js/stdlib/ts/ALGO.ts

```diff
@@ -1217,17 +1217,16 @@ const makeTransferTxn = (
 
 const transfer = async (
   from: Account,
-  to: Account,
+  to: Address,
   value: any,
   token?: Token,
   opts?: TransferOpts,
 ): Promise<RecvTxn> => {
   const sender = from.networkAccount;
-  const receiver = extractAddr(to);
```

Comment:
You should still use this function, because we want to allow people to pass `Account`s, since they can be turned into addresses